### PR TITLE
updated Redis to Redis®

### PR DESCRIPTION
**Pull request by dragonlancewu**
Fixes: #55

### CODE REVIEW

**Review by jlprat — DISMISSED** [review]
LGTM

**Comment by lornajane** [comment]
The changes are good, but is this all of them? @TibsAtWork do we need do adjust the boilerplate footer text also?

**Comment by runwuf** [comment]
@lornajane @TibsAtWork I've updated & added boilerplate footer text as well, also ripgrep checked this is all of `Redis`

File: redis/nodejs/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-### Redis node.js Example
+### Redis® node.js Example
 
-This example uses the [ioredis](https://www.npmjs.com/package/ioredis) library to connect to Redis, write a key/value and read it out again.
+This example uses the [ioredis](https://www.npmjs.com/package/ioredis) library to connect to Redis®, write a key/value and read it out again.
```
Comment (TibsAtWork) [inline redis/nodejs/README.md:4]: This file is missing the acknowledgement text for Redis (the `Trademark` section)

File: terraform/README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ It is used to demonstrate migration path from TF 0.12 with Aiven provider 1.3.5
       │      │                           │     │     │                │         │
       │      │                         ┌─┘     │     │                │      ┌──┴──────┐
       │      │                         │       │     │                │      │         │
-      │      │                         │       │     └────Metrics─────┼──────┤  Redis  │
+      │      │                         │       │     └────Metrics─────┼──────┤  Redis® │
```
Comment (TibsAtWork) [inline terraform/README.md:32]: If we're putting the ® after Redis, it also needs an asterisk. But we haven't put ® marks for the other products in this diagram, nor are there acknowledgements of their registration status.

Also, separate point, do we want Kibana at the top, or OpenSearch Dashboards? I think the terraform code actually refers to OpenSearch Dashboards now.

For the moment, I think it's best to just sort out Redis, as you've been doing, and I'll raise a new issue to look at checking that this page is up-to-date.

**Comment by dragonlancewu** [comment]
Thanks @TibsAtWork for reviewing my work, all fixed as per your suggestion.

**Review by TibsAtWork — COMMENTED** [review]
I think this is indeed all the places that need fixing, and there's just one more minor change. Thanks.

File: terraform/README.md [hunk 3]
```diff
@@ -345,7 +345,7 @@ resource "aiven_database" "pg_db" {
 
 ```
 
-**Redis**
+**Redis®**
```
[the remark this answers is not in the document]
Comment (dragonlancewu) [reply terraform/README.md]: @TibsAtWork thank you for catching this, I have fixed it.

**Pull request merged**